image/postgres/Dockerfile.envsubst

```
@@ -26,15 +26,23 @@ COPY scripts/docker-entrypoint.sh /usr/local/bin/
 COPY scripts/init-entrypoint.sh /usr/local/bin/
 COPY --from=extracted_bundle /bundle/postgres.rpm /bundle/postgres-libs.rpm /bundle/postgres-server.rpm /bundle/postgres-contrib.rpm /tmp/
 
-RUN groupadd -g 70 postgres && \
+RUN microdnf upgrade -y --nobest && \
+    microdnf install -y shadow-utils && \
+    groupadd -g 70 postgres && \
     adduser postgres -u 70 -g 70 -d /var/lib/postgresql -s /bin/sh && \
     rpm --import RPM-GPG-KEY-PGDG-13 && \
-    dnf upgrade -y --nobest && \
-    dnf install -y \
+    microdnf install -y \
         ca-certificates libicu systemd-sysv \
         glibc-locale-source glibc-langpack-en \
-        /tmp/postgres-libs.rpm /tmp/postgres-server.rpm /tmp/postgres.rpm /tmp/postgres-contrib.rpm && \
-    dnf clean all && \
+        perl-libs libxslt && \
+    rpm -i /tmp/postgres-libs.rpm /tmp/postgres-server.rpm /tmp/postgres.rpm /tmp/postgres-contrib.rpm && \
+    # The removal of /usr/share/zoneinfo from UBI minimal images is intentional.
+    # After building the image, the image is reduced in size as much as possible,
+    # and the /usr/share/zoneinfo directory is purged as it saves space
+    # in the final distribution of the image.
```

This comment is quite confusing. You are not removing `/usr/share/zoneinfo`, are you?

In my opinion, this comment should say something like
```suggestion
    # Restore /usr/share/zoneinfo that's empty in ubi-minimal because _X_ needs it. @janisz please write what that X is.
```